src/OrbitGl/TrackManagerTest.cpp

```diff
@@ -154,4 +154,25 @@ TEST_F(TrackManagerTest, NonVisibleTracksAreNotInTheList) {
   EXPECT_EQ(kNumTracks - 1, track_manager_->GetVisibleTracks().size());
 }
 
+TEST_F(TrackManagerTest, TrackTypeVisibilityAffectsVisibleTrackList) {
+  CreateAndFillTracks();
+
+  track_manager_->SetTrackTypeVisibility(Track::Type::kSchedulerTrack, false);
+  track_manager_->UpdateTracksForRendering();
+  EXPECT_EQ(kNumTracks - 1, track_manager_->GetVisibleTracks().size());
+
+  track_manager_->SetTrackTypeVisibility(Track::Type::kThreadTrack, false);
+  track_manager_->UpdateTracksForRendering();
+  EXPECT_EQ(kNumTracks - 2, track_manager_->GetVisibleTracks().size());
```

Comment:
I would like to have some kNumThreadTracks that can vary here. I would like also in the future to have a few ThreadTracks with different names to test the filter.

So, I will replace kNumTracks - 2 for kNumTracks - 1 - kNumThreadTracks

Reply:
Good suggestion, thanks, added (also not yet pushed).